instruments/astrooda_isgri/astrooda_isgri.inc

```diff
@@ -256,7 +256,7 @@ function astrooda_isgri($form, &$form_state)
     '#prefix' => '<div class="row">'
   );
 
-  $form['E1_keV']['#default_value'] = 20;
+  $form['E1_keV']['#default_value'] = 25;
```

Comment:
This is not needed, and it's actually not the right way to set a default value, just the change that you applied in `.install` file should do the job.

Reply:
Also, what is missing is the setting of the `default_value` for `E1_keV` (as well as for `E2_keV`, and I think also `detection_threshold` if I am not wrong) reading it from the `instrument_defaults` var.